cfme/common/vm.py

```diff
@@ -301,9 +300,9 @@ def find_quadicon(
                 # TODO implement as navigate_to when cfme.infra.virtual_machines has destination
                 navigate_to(self, 'All')
             elif self.is_vm:
-                self.provider.load_all_provider_vms()
+                navigate_to(self, 'AllForProvider', use_resetter=False)
```

Comment:
any VM View from provider doesn't provide Archived/Orphaned VMs as they are not related to any provider any more.